impeller/entity/inline_pass_context.cc

```diff
@@ -58,22 +58,26 @@ bool InlinePassContext::EndPass() {
   }
 
   if (command_buffer_) {
-    if (!command_buffer_->EncodeAndSubmit(pass_)) {
+    if (!pass_->EncodeCommands()) {
```

Comment:
This is kind of weird now, it isn't obvious why there is a `if (command_buffer_)` check here.  Is that still necessary?

Reply:
Should be handled by isActive check, added D_CHECK